src/sentry/api/endpoints/organization_tags.py

```diff
@@ -35,8 +43,7 @@ def get(self, request: Request, organization) -> Response:
                     filter_params["start"],
                     filter_params["end"],
                     use_cache=request.GET.get("use_cache", "0") == "1",
-                    # Defaults to True, because the frontend caches these tags globally
-                    include_transactions=request.GET.get("include_transactions", "1") == "1",
```

Comment:
Just double checking that the frontend is not using `include_transactions` anywhere? If it was we would need to handle it sending `include_transactions=0`

Reply:
AFAICT, at least for this endpoint there are no usages of `include_transactions` on the frontend. (A similar endpoint, `organization_tagkey_values` _does_ use this though)